packages/zoltan2/src/algorithms/color/Zoltan2_AlgHybridGMB.hpp

```diff
@@ -0,0 +1,458 @@
+#ifndef _ZOLTAN2_ALGHYBRIDGMB_HPP_
+#define _ZOLTAN2_ALGHYBRIDGMB_HPP_
+
+#include <vector>
+#include <unordered_map>
+#include <iostream>
+#include <queue>
+#include <unistd.h>
```

Comment:
This is not a Standard C++ header file.  People like to include this file to get the definition of `NULL`.  However, we have C++11 now; use `nullptr` instead.

Reply:
The underlying cause of this include has been removed, so I have removed the include.